utils/amount/capacity.go

```diff
@@ -0,0 +1,11 @@
+package amount
+
+import "math"
+
+func CkbToShannon(amount uint64) uint64 {
+	return amount * exponent(10, 8)
```

Comment:
suggest to hardcode here:
```suggestion
	return amount * uint64(100000000)
```